bin/qmk

```diff
@@ -35,7 +35,7 @@ def _check_modules(requirements):
 
             if not find_spec(module['import']):
                 print('Could not find module %s!' % module['name'])
-                print('Please run `python3 -m pip install -r %s` to install required python dependencies.' % str(qmk_dir / requirements))
+                print('Please run `python3 -m pip install -r %s` to install required python dependencies.' % (qmk_dir / requirements,))
```

Comment:
```suggestion
                print('Please run `python3 -m pip install -r %s` to install required python dependencies.' % (qmk_dir / requirements))
```

Reply:
The trailing comma is deliberate. It's a habit I developed to avoid confusion between tuples and parens as a math operator.